.gitignore

```diff
@@ -1,20 +1,7 @@
 .vs/
-Generator/bin/
-Generator/obj/
-GeneratorLib/bin/
-GeneratorLib/obj/
-GeneratorUnitTests/bin/
-GeneratorUnitTests/obj/
-InstallerNuget/bin/
-InstallerNuget/obj/
-glTFLoader/bin/
-glTFLoader/obj/
-glTFLoaderUnitTests/bin/
-glTFLoaderUnitTests/obj/
-glTFLoader_Shared/bin/
-glTFLoader_Shared/obj/
+[Bb]in/
+[Oo]bj/
```

Comment:
Can bin/obj have first letter capitalized?

Reply:
It can on Windows. In practice probably never. I just copied this from an existing `.gitignore`.

Reply:
Given what you replaced, it probably isn't necessary.